tests/translator/input/connector_function_to_location.yaml

```diff
@@ -0,0 +1,22 @@
+Resources:
+  MyFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Runtime: nodejs14.x
+      Handler: index.handler
+      InlineCode: |
+        const AWS = require('aws-sdk');
+        exports.handler = async (event) => {
+          console.log(JSON.stringify(event));
+        };
+
+  MyConnector:
+    Type: AWS::Serverless::Connector
+    Properties:
+      Source:
+        Id: MyFunction
+      Destination:
+        Arn: arn:aws:geo:us-east-1:123123123123:place-index/explore.place
+        Type: AWS::Location::PlaceIndex
```

Comment:
Can you also add another connector that references an `Id` to a `AWS::Location::PlaceIndex` in the same template as well?

Reply:
Absolutely - done